<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configuration</class>
 <widget class="QDialog" name="Configuration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>415</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1098</width>
    <height>880</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(45, 45, 45);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="styleSheet">
         <string notr="true">QFrame{
	background-color: rgb(35, 35, 35);
	border-radius:5px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../resource/asset.qrc">:/icons/asset/settings.svg</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel{
	color: rgb(255, 255, 255);
}</string>
              </property>
              <property name="text">
               <string>Configurations</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_minimize">
              <property name="minimumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>17</width>
                <height>17</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	background-color: rgb(85, 255, 127);
	border:none;
	border-radius: 8px;
}

QPushButton:hover{	
	background-color: rgba(85, 255, 127,150);
	
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_maximize">
              <property name="minimumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>17</width>
                <height>17</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	
	background-color: rgb(255, 170, 0);
	border:none;
	border-radius: 8px;
}

QPushButton:hover{	
	background-color: rgba(255, 170, 0,150);
	
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_close">
              <property name="minimumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>17</width>
                <height>17</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	background-color: rgb(255, 0, 0);
	border:none;
	border-radius: 8px;
}

QPushButton:hover{	
	background-color: rgba(255, 0, 0,150);
	
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QFrame" name="frame_6">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <widget class="QLabel" name="firstname_25">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>451</width>
                 <height>331</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>10</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	background-color: rgb(45, 45, 45);
	border-radius: 10px;
	padding-left:10px;
	padding-top:5px;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
              </widget>
              <widget class="QLineEdit" name="camera_ip">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>80</y>
                 <width>421</width>
                 <height>50</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit{
    background-color: rgb(35,35,35);
	color: rgb(255, 255, 255);
	border: 2px solid  rgb(45, 45, 45);
	border-radius:15px;
	padding-left: 60px;
}

QLineEdit:hover{
	border:2px solid rgb(35, 35, 35);
}

QLineEdit:focus{
	border:2px solid rgb(255, 255, 255);
}</string>
               </property>
               <property name="placeholderText">
                <string>Camera Id/IP ?</string>
               </property>
               <property name="clearButtonEnabled">
                <bool>true</bool>
               </property>
              </widget>
              <widget class="QPushButton" name="btn_insert">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>150</y>
                 <width>131</width>
                 <height>41</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(45, 45, 45);
	border:none;
background-color: rgb(35, 35, 35);
	border-radius: 10px;
}

QPushButton:hover{
	border-radius: 20px;
}

QPushButton:pressed{
	border: 2px solid rgb(255,255,255);	
}</string>
               </property>
               <property name="text">
                <string>Insert</string>
               </property>
               <property name="icon">
                <iconset resource="../resource/asset.qrc">
                 <normaloff>:/icons/asset/plus.svg</normaloff>:/icons/asset/plus.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
              <widget class="QPushButton" name="btn_update">
               <property name="geometry">
                <rect>
                 <x>150</x>
                 <y>150</y>
                 <width>141</width>
                 <height>41</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(45, 45, 45);
	border:none;
background-color: rgb(35, 35, 35);
	border-radius: 10px;
}

QPushButton:hover{
	border-radius: 20px;
}

QPushButton:pressed{
	border: 2px solid rgb(255,255,255);	
}</string>
               </property>
               <property name="text">
                <string>Update</string>
               </property>
               <property name="icon">
                <iconset resource="../resource/asset.qrc">
                 <normaloff>:/icons/asset/repeat.svg</normaloff>:/icons/asset/repeat.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
              <widget class="QPushButton" name="btn_delete">
               <property name="geometry">
                <rect>
                 <x>300</x>
                 <y>150</y>
                 <width>131</width>
                 <height>41</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(45, 45, 45);
	border:none;
background-color: rgb(35, 35, 35);
	border-radius: 10px;
}

QPushButton:hover{
	border-radius: 20px;
}

QPushButton:pressed{
	border: 2px solid rgb(255,255,255);	
}</string>
               </property>
               <property name="text">
                <string>Delete</string>
               </property>
               <property name="icon">
                <iconset resource="../resource/asset.qrc">
                 <normaloff>:/icons/asset/trash-2.svg</normaloff>:/icons/asset/trash-2.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
              <widget class="QLabel" name="label_28">
               <property name="geometry">
                <rect>
                 <x>30</x>
                 <y>90</y>
                 <width>31</width>
                 <height>30</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(35,35,35);</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="../resource/asset.qrc">:/icons/asset/video.svg</pixmap>
               </property>
              </widget>
              <widget class="QLabel" name="label_notification">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>210</y>
                 <width>421</width>
                 <height>101</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <family>MS Shell Dlg 2</family>
                 <pointsize>10</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	background-color: rgb(35, 35, 35);
	border-radius: 10px;
}</string>
               </property>
               <property name="text">
                <string>Notification</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
              <widget class="QLineEdit" name="camera_reference">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>20</y>
                 <width>271</width>
                 <height>47</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>47</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>45</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit{
    background-color: rgb(35,35,35);
	color: rgb(255, 255, 255);
	border: 2px solid  rgb(45, 45, 45);
	border-radius:15px;
	padding-left: 8px;
}

QLineEdit:hover{
	border:2px solid rgb(35, 35, 35);
}

QLineEdit:focus{
	border:2px solid rgb(255, 255, 255);
}</string>
               </property>
               <property name="placeholderText">
                <string>Camera Id ?</string>
               </property>
               <property name="clearButtonEnabled">
                <bool>true</bool>
               </property>
              </widget>
              <widget class="QPushButton" name="btn_search">
               <property name="geometry">
                <rect>
                 <x>290</x>
                 <y>20</y>
                 <width>141</width>
                 <height>45</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>45</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(45, 45, 45);
	border:none;
background-color: rgb(35, 35, 35);
	border-radius: 10px;
}

QPushButton:hover{
	border-radius: 20px;
}

QPushButton:pressed{
	border: 2px solid rgb(255,255,255);	
}</string>
               </property>
               <property name="text">
                <string>Search</string>
               </property>
               <property name="icon">
                <iconset resource="../resource/asset.qrc">
                 <normaloff>:/icons/asset/search.svg</normaloff>:/icons/asset/search.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/asset.qrc"/>
 </resources>
 <connections/>
</ui>
